css/styles.css

```diff
@@ -78,20 +96,53 @@ header .logo {
 }
 
 .container img {
-    float: left;
-    margin: 20px 0 0 0;
 }
 
 .container p {
     background-color: #9C4D13;
     color: #fff;
-    padding: 5px 0 5px 10px;
-    width: 580px;
 }
 
 /***** ----- Footer ----- *****/
 
 footer {
     background-color: #C6621A;
+    color: #fff;
     margin-top: 20px;
+}
+
+.column {
+    float: left;
+    width: 25%;
+}
+
+.column:last-child {
+    width: 50%;
+}
+
+.column h3 {
+    border-bottom: 1px solid #fff;
+    margin: 15px 10px 15px 0;
```

Comment:
Instead of `10px` for the margin right, add padding to the 2 firsts columns `padding: 10px 10px 0 0` and it will create the necessary space